src/frontend/custom-sequencer.ts

```diff
@@ -0,0 +1,29 @@
+const Sequencer = require("@jest/test-sequencer").default;
+
+/** A sequencer that ensures that ensures that the slowest tests
+ * all land on different shards. */
+class CustomSequencer extends Sequencer {
+  shard(tests, { shardIndex, shardCount }) {
+    // Tests by order of increasing duration. Update by looking at test duration.
+    // Does not need to be exact.
+    const testFilesByDuration = [
```

Comment:
would it be worth to build this array dynamically? List files in ordner x with NodeJS?

Reply:
Actually I don't need a list of all the files, I just need the files that take longer to run. What's important here is the order of the files.